app/Http/Controllers/Entity/AssetController.php

```diff
@@ -99,14 +99,21 @@ protected function storeFile(StoreEntityAsset $request, Campaign $campaign, Enti
         $service = app()->make(EntityFileService::class);
 
         try {
-            $file = $service
+            $files = $service
                 ->entity($entity)
                 ->campaign($campaign)
                 ->upload($request, 'file');
```

Comment:
DOes this work? The field is renamed files[] (correct) but the code still checks for `file`?